src/ui/blockUi-timeBased.js

```diff
@@ -28,51 +28,42 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
             onMediaEnded: null
         },
         invokers: {
-            updateMediaPlayer: {
-                "funcName": "sjrk.storyTelling.blockUi.timeBased.updateMediaPlayer",
-                "args": ["{that}.dom.mediaPlayer", "{arguments}.0"]
+            updateMediaPreview: {
+                "funcName": "sjrk.storyTelling.blockUi.timeBased.updateMediaPreview",
+                "args": ["{that}.dom.mediaPreview", "{arguments}.0"]
             },
             stopMediaPlayer: {
                 func: "{that}.events.onMediaPlayerStop.fire",
                 args: ["{that}"]
             },
             playMediaPlayer: {
                 "funcName": "sjrk.storyTelling.blockUi.timeBased.playMediaPlayer",
-                "args": ["{that}.dom.mediaPlayer"]
+                "args": ["{that}.dom.mediaPreview"]
             }
         },
+        // describes the relationship of DOM events to associated Infusion events
+        mediaPlayerEventMapping : {
+            canplay: "{that}.events.onMediaReady.fire",
+            durationchange: "{that}.events.onMediaDurationChange.fire",
+            ended: "{that}.events.onMediaEnded.fire",
+            play: "{that}.events.onMediaPlay.fire"
+        },
```

Comment:
Another option here would be to just have the Infusion event names on the right hand side. In that case `sjrk.storyTelling.blockUi.timeBased.registerMediaPlayerEvents` would need to take in `that` and would construct the output even firer like `that.events[eventName].fire'